lib/einhorn/event/connection.rb

```diff
@@ -57,13 +57,13 @@ def unsubscribe(tag)
     end
 
     def register!
-      log_info("client connected")
+      log_debug("client connected")
       Einhorn::Event.register_connection(self, @socket.fileno)
       super
     end
 
     def deregister!
-      log_info("client disconnected") if Einhorn::TransientState.whatami == :master
+      log_debug("client disconnected") if Einhorn::TransientState.whatami == :master
```

Comment:
I honestly don't understand the scenario where you'd want this, but my inclination is to not change it since it seems like it can only strictly increase log noise, and I'm trying to get out of that business.